src/lib/stitching/gravity/schema.ts

```diff
@@ -61,6 +61,13 @@ export const executableGravitySchema = () => {
     duplicatedTypes.push("MarketingCollectionGroup")
     duplicatedTypes.push("MarketingCollectionCategory")
   }
+
+  if (config.USE_UNSTITCHED_ARTIST_SERIES_SCHEMA) {
```

Comment:
Eventually when the types are all cleaned up in Gravity - this snippet can go away, but for now we need it - it says to not import these types from Gravity (as we've defined them here in MP).